test.bat

```diff
@@ -1,39 +0,0 @@
-@echo off

```

Comment:
Did you mean to remove this file?
Instead of updating it to call `build.ps1` with the tests target?

Reply:
As discussed on a call, this was due to making things the same as they are in other repositories that use Chocolatey.Cake.Recipe.  I will open up a follow up issue to add an entry bat file for running tests.